lib/rucio/daemons/tracer/kronos.py

```diff
@@ -409,11 +409,7 @@ def kronos_dataset(once: bool = False, dataset_queue: Queue = None, sleep_time:
 
 def run_once_kronos_dataset(heartbeat_handler: HeartbeatHandler, dataset_queue: Queue, **kwargs):
     _, _, logger = heartbeat_handler.live()
-    dataset_wait = config_get_int('tracer-kronos', 'dataset_wait')
```

Comment:
To me it looks like the correct fix would be to move 
```
    start = datetime.now()
```
between lines 392 and 293 above and pass it to run_once_kronos_dataset as an argument